simulariumio/data_objects/display_data.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class DisplayData:
+    name: str
+    radius: float
+    display_type: str
```

Comment:
instead of str, this should be DISPLAY_TYPE?

Reply:
The fields in DISPLAY_TYPE are strings, so I was thinking if you do `DisplayData(display_type=DISPLAY_TYPE.SPHERE)` the DisplayData object you create has a string display_type = "SPHERE". I'm not as familiar with Python objects, I can make the type DISPLAY_TYPE? seems like then I'd need each display type to be a class deriving from DISPLAY_TYPE?

Reply:
I was picturing something like an Enum : https://docs.python.org/3/library/enum.html
It would be nice to be able to treat DISPLAY_TYPE as a type.  Something to think about anyway...
Also you might be able to do 
`display_type : DISPLAY_TYPE.SPHERE | DISPLAY_TYPE.FIBER...etc`
but 
`display_type: DISPLAY_TYPE` would be nicer

Reply:
aha yeah I didn't realize I could do enums like that, it's updated now